flujo/application/core/executor_core.py

```
@@ -2764,8 +2767,56 @@ def _is_mock(obj: Any) -> bool:
+                    from flujo.utils.json_normalizer import normalize_to_json_obj as _norm
+                    from flujo.agents.repair import DeterministicRepairProcessor as _DR
+
+                    # Normalize strings into JSON when possible
+                    if isinstance(processed_output, str):
+                        maybe_obj = _norm(processed_output)
+                        if isinstance(maybe_obj, (dict, list)):
+                            processed_output = maybe_obj
+                        else:
+                            try:
+                                cleaned = await _DR().process(processed_output)
+                            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)